### DISCUSSION TITLE
EOL for Oracle Linux OS is not aligned with the EOL date mentioned in official site

### DISCUSSION THREAD

**Discussion by s-reddy1498** [aquasecurity/trivy#7479] [Bugs]
### Description

For Oracle we have provided EOL for version 7 as 23 Aug 2024, but in EOL site it is provided as 31 Dec 2024. This might effect the fix version and remediation info for a CVE. 

i.e. This might cause an issue if we have any CVE for which the fix version should be upgrade OS to oracle Linux of version 7, but will be shown as upgrade to oraclelinux:8 version as the version 7 has reached EOL according to date mention in trivy detector.

### Desired Behavior

The EOL date should be in-lined. 

### Actual Behavior

Mismatch in the EOL date, which might cause an issue if we have any CVE in oraclelinux:6 OS for which the fix version should be upgrade OS to oracle Linux of version 7, but will be shown as upgrade to oraclelinux:8 version as the version 7 has reached EOL according to date mention in Trivy detector.

![image](https://github.com/user-attachments/assets/b9f6811f-922a-4f30-bbc8-a4c236cd7508)

![image](https://github.com/user-attachments/assets/2ff66d44-bc99-4181-b795-9aecbc99dc74)

### Reproduction Steps

```bash
1. Navigate to Trivy OS package detector (i.e., https://github.com/aquasecurity/trivy/blob/main/pkg/detector/ospkg/oracle/oracle.go)
2. The EOL date for version 7 is provided as 23 Aug 2024
3. The EOL date in website is mentioned as 31 Dec 2024. (ref: https://endoflife.date/oracle-linux)
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
N/A
```

### Operating System

Oracle Linux

### Version

```bash
N/A
```

### Checklist

- [X] Run `trivy clean --all`
- [X] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by DmitriyLewen** [comment]
EOL dates have been updated in #7480
